index.rst

```diff
@@ -48,14 +48,261 @@
+- Azimuth
+- Camera rotator angle
+
+.. note::
+    Here we are only concerned with the LUTs that are used during exposures. The M1M3 also has a dynamic LUT that are used during slewing, but it only affects the image quality indirectly (if the system doesn’t settle completely before exposure starts).
+
+#########
+Objective
+#########
+
+The implementation details we cover in this tech note are mostly about conventions, i.e., if we make different choices, it probably won't matter much to the performance of the AOS.
+The main objective of this tech note is to make the implementation of the LUTs as consistent as possible across the components, while accommodating the unique design and functionalities of each.
+This is crucial for avoiding confusions later, both during commissioning of the AOS, and in operations.
+
+.. Important::
+
+  It is important to acknowledge that the personel who would be using this system will have a diverse background, ranging from scientists, engineers, to software developers and telescope operators. As the result, people from different background may have different opinions on what is better and more intuitive.
+  What we do in this tech note is that we try to balance the different perspectives.
+  The more important thing is to have a standard, not what standard.
+
+
+###############################
+Elevation angle vs zenith angle
+###############################
+
+We should use elevation angle for elevation, instead of zenith angle.
+The elevation angle is zero when the telescope points at horizon, and 90 deg when the telescope points at zenith.
+
+It is OK if a subsystem or an analysis notebook uses zenith angle internally, if the variable name is self-explanatory. The input variable to the gravity component of the LUTs should be elevation angle. The output of data analysis, which would be passed to software developers to implement, should also use elevation angle.
+
+The gravity components of the LUTs have different elevation ranges,
+
+- [-8.5, 98.5] deg for M1M3. Occasionally M1M3 may go out of the [0, 90] deg range, and we want to make sure the glass mirror can still be supported properly. Hence the 8.5 deg margin on both sides.
+- [-270, 90] deg for M2. In normal operations, M2 elevation will never go beyond the [0, 90] deg range. But M2 has a cart. For testing and engineering purposes, it needs to be able to be rotated by 360 deg. Beyond the [0, 90] deg range, glass safety is still the concern, but not image quality.
+- [0, 90] deg for the M2 and camera hexapods. The hexapods may go out of the [0, 90] deg range. But we won't be imaging outside of that range. Unlike the mirrors, where outside of [0, 90] deg proper LUT forces are still needed to counter gravity, for hexapods, we do not have to compensate position offset due to gravity. If the elevation angle goes smaller than 0 deg, we will use LUT values for 0 deg. If it goes larger than 90 deg we will use LUT values for 90 deg.
+
```

Comment:
Should we add here the origin of the elevation angle value for each components?

Reply:
Can you say more about what you mean by "origin"? The elevation angle is uniquely determined once we know the optical axis orientation and the gravity vector. It is not coordinate system specific, right?

Reply:
I meant to define where the information of the elevation comes from (TMA or internal inclinometer).

Reply:
Thanks. See the new paragraph I added to the end of Sec. 3.